<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="2" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-12-17T08:21:09 IST" name="Failed suite [Default suite]" finished-at="2022-12-17T08:21:41 IST" duration-ms="32679">
    <groups>
    </groups>
    <test started-at="2022-12-17T08:21:09 IST" name="Default test(failed)" finished-at="2022-12-17T08:21:41 IST" duration-ms="32679">
      <class name="com.automationtesting.testngsession.OrangHrmTest">
        <test-method is-config="true" signature="propertiesData()[pri:0, instance:com.automationtesting.testngsession.OrangHrmTest@169e6180]" started-at="2022-12-17T08:21:09 IST" name="propertiesData" finished-at="2022-12-17T08:21:09 IST" duration-ms="9" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- propertiesData -->
        <test-method is-config="true" signature="basicSetup()[pri:0, instance:com.automationtesting.testngsession.OrangHrmTest@169e6180]" started-at="2022-12-17T08:21:09 IST" name="basicSetup" finished-at="2022-12-17T08:21:11 IST" duration-ms="2042" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- basicSetup -->
        <test-method is-config="true" signature="launchUrl()[pri:0, instance:com.automationtesting.testngsession.OrangHrmTest@169e6180]" started-at="2022-12-17T08:21:11 IST" name="launchUrl" finished-at="2022-12-17T08:21:14 IST" duration-ms="3340" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchUrl -->
        <test-method is-config="true" signature="login()[pri:0, instance:com.automationtesting.testngsession.OrangHrmTest@169e6180]" started-at="2022-12-17T08:21:14 IST" name="login" finished-at="2022-12-17T08:21:15 IST" duration-ms="620" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="RegisterEmployee()[pri:2, instance:com.automationtesting.testngsession.OrangHrmTest@169e6180]" started-at="2022-12-17T08:21:15 IST" name="RegisterEmployee" finished-at="2022-12-17T08:21:23 IST" duration-ms="8844" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Unable to add employee Test Failed expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Unable to add employee Test Failed expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at com.automationtesting.testngsession.OrangHrmTest.RegisterEmployee(OrangHrmTest.java:96)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RegisterEmployee -->
        <test-method is-config="true" signature="logOut()[pri:0, instance:com.automationtesting.testngsession.OrangHrmTest@169e6180]" started-at="2022-12-17T08:21:23 IST" name="logOut" finished-at="2022-12-17T08:21:25 IST" duration-ms="1530" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOut -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.automationtesting.testngsession.OrangHrmTest@169e6180]" started-at="2022-12-17T08:21:25 IST" name="tearDown" finished-at="2022-12-17T08:21:41 IST" duration-ms="16264" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.automationtesting.testngsession.OrangHrmTest -->
    </test> <!-- Default test(failed) -->
  </suite> <!-- Failed suite [Default suite] -->
</testng-results>
